modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+# Copyright 2020 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+
+# Reference all required bash variables prior to running. Due to 'nounset', if
+# a caller fails to export the following expected environmental variables, this
+# script will fail immediately rather than partially succeeding.
+echo "Cloud SQL Instance Connection string: ${SQL_INSTANCE_CONN_STRING}"
```

Comment:
If you want to use an env var instead of string interpolation (coming from a TF var), then omit the curly brackets. I see this for quite a few variables in this file.

Reply:
Done